compiler-core/src/javascript/typescript.rs

```diff
@@ -364,69 +346,48 @@ impl<'a> TypeScriptGenerator<'a> {
         }
     }
 
-    fn definition(
-        &mut self,
-        definition: &'a TypedDefinition,
-        imports: &mut Imports<'_>,
-    ) -> Vec<Document<'a>> {
-        match definition {
-            Definition::TypeAlias(TypeAlias {
-                alias,
-                publicity,
-                type_,
-                ..
-            }) if publicity.is_importable() => vec![self.type_alias(alias, type_)],
-            Definition::TypeAlias(TypeAlias { .. }) => vec![],
+    fn definitions(&mut self, imports: &mut Imports<'_>) -> Vec<Document<'a>> {
```

Comment:
Looks like this vec is turned immediately into an iterator, so could return one here? Up to you.

Reply:
I couldn't figure out a nice way to make that an iterator from the start, chaining things messed with the borrow checker